### PR TITLE
[zh] sync rbac-good-practices autoscaling

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 1]
```diff
@@ -369,6 +369,20 @@ mutating webhooks, also mutate admitted objects.
...
+### Namespace modification
+
+Users who can perform **patch** operations on Namespace objects (through a namespaced RoleBinding to a Role with that access) can modify
+labels on that namespace. In clusters where Pod Security Admission is used, this may allow a user to configure the namespace
+for a more permissive policy than intended by the administrators.
+For clusters where NetworkPolicy is used, users may be set labels that indirectly allow
+access to services that an administrator did not intend to allow.
+-->
+### 命名空间修改 {#namespace-modification}
+可以对命名空间对象执行 **patch** 操作权限的用户（通过命名空间内的 RoleBinding 关联到具有该权限的角色），
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:382]:
```suggestion
可以对命名空间对象执行 **patch** 操作的用户（通过命名空间内的 RoleBinding 关联到具有该权限的 Role），
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 2]
```diff
@@ -266,20 +266,14 @@ If scaling workloads isn't enough to meet your needs, you can also scale your cl
...
 
 <!--
-Both scalers work by watching for pods marked as _unschedulable_ or _underutilized_ nodes and then adding or
-removing nodes as needed.
+Read [cluster autoscaling](/docs/concepts/cluster-administration/cluster-autoscaling/)
+for more information.
 -->
-这两种扩缩器的工作原理都是通过监测节点上被标记为 **unschedulable** 或 **underutilized** 的 Pod 数量，
-然后根据需要增加或移除节点。
+阅读[集群自动伸缩](/zh-cn/docs/concepts/cluster-administration/cluster-autoscaling/)了解更多信息。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:276]:
```suggestion
阅读[集群自动扩缩](/zh-cn/docs/concepts/cluster-administration/cluster-autoscaling/)了解更多信息。
```

File: content/zh-cn/docs/concepts/workloads/autoscaling.md [hunk 3]
```diff
@@ -289,9 +283,11 @@ removing nodes as needed.
...
 - [Resize Container Resources In-Place](/docs/tasks/configure-pod-container/resize-container-resources/)
 - [Autoscale the DNS Service in a Cluster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/)
+- Learn about [cluster autoscaling](/docs/concepts/cluster-administration/cluster-autoscaling/)
 -->
 - 了解有关横向扩缩的更多信息
   - [扩缩 StatefulSet](/zh-cn/docs/tasks/run-application/scale-stateful-set/)
   - [HorizontalPodAutoscaler 演练](/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/)
 - [调整分配给容器的 CPU 和内存资源](/zh-cn/docs/tasks/configure-pod-container/resize-container-resources/)
 - [自动扩缩集群 DNS 服务](/zh-cn/docs/tasks/administer-cluster/dns-horizontal-autoscaling/)
+- 了解[集群自动伸缩]((/zh-cn/docs/concepts/cluster-administration/cluster-autoscaling/))
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/autoscaling.md:293]:
```suggestion
- 了解[集群自动扩缩]((/zh-cn/docs/concepts/cluster-administration/cluster-autoscaling/))
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**